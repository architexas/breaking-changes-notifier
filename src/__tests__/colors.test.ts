import colors from '../colors';

describe('console colors file', () => {
  it('should consist of the possible colors to print', () => {
    expect(colors).toMatchSnapshot();
  });
});
