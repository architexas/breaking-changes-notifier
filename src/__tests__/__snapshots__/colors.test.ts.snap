// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`console colors file should consist of the possible colors to print 1`] = `
Object {
  "Blink": "[5m",
  "Bright": "[1m",
  "Dim": "[2m",
  "Hidden": "[8m",
  "Reset": "[0m",
  "Reverse": "[7m",
  "Underscore": "[4m",
  "bg": Object {
    "Black": "[40m",
    "Blue": "[44m",
    "Crimson": "[48m",
    "Cyan": "[46m",
    "Green": "[42m",
    "Magenta": "[45m",
    "Red": "[41m",
    "White": "[47m",
    "Yellow": "[43m",
  },
  "fg": Object {
    "Black": "[30m",
    "Blue": "[34m",
    "Crimson": "[38m",
    "Cyan": "[36m",
    "Green": "[32m",
    "Magenta": "[35m",
    "Red": "[31m",
    "White": "[37m",
    "Yellow": "[33m",
  },
}
`;
